packages/server/package.json

```diff
@@ -6,6 +6,7 @@
     "keywords": [],
     "author": "Hedera Smart Contracts Team",
     "dependencies": {
+        "@ethersproject/asm": "^5.7.0",
```

Comment:
Q: why is this needed on the server side package?

Reply:
Leftover from tests. Removed.